cornflow/endpoints/action.py

```diff
@@ -0,0 +1,39 @@
+"""
+
+"""
+# Import from libraries
+from flask_apispec import marshal_with, doc
+from flask_apispec.views import MethodResource
+
+# Import from internal modules
+from .meta_resource import MetaResource
+from ..models import ActionModel
+from ..schemas.action import ActionsResponse
+from ..shared.authentication import Auth
+from ..shared.const import ADMIN_ROLE, SERVICE_ROLE
+
+
+class ActionListEndpoint(MetaResource, MethodResource):
+    ROLES_WITH_ACCESS = [ADMIN_ROLE]
+    DESCRIPTION = "Endpoint which allows to get the actions defined in the application"
+
+    def __init__(self):
+        super().__init__()
+        self.model = ActionModel
+        self.query = ActionModel.get_all_objects
+        self.primary_key = "id"
```

Comment:
They used to have more sense when we used more the MetaResource class, but since we started using it less they may not be needed but I always put them just in case